idex.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Awesome Books</title>
+</head>
+<body>
+    <section>
+        <main class="main">
+            <h1>All awesome books</h1>   
+            <div class="books-list"></div>
+            <hr>
+            <h1>Add a new book</h1>
+            <form class="add-form">
+                <label for="title" hidden>Title:</label>
+                <input type="text" class="title" required placeholder="Title"/>
+                <label for="author" hidden>Author:</label>
+                <input type="text" class="author" required placeholder="Author"/>
+                <button type="submit" class="form-btn">Add</button>
+            </form>            
```

Comment:
- [ ] [OPTIONAL] I noticed that you're using the for attribute on the labels, but you're not using the name attribute on the input elements.

The name attribute is important for two reasons:

It allows you to reference the input elements in JavaScript.
It allows you to submit the values of the input elements when the form is submitted.
Without the name attribute, the values of the input elements will not be submitted when the form is submitted.

I recommend that you add the name attribute to the input elements. For example, you could change the code to the following:

```html

<form class="add-form">
<label for="title">Title:</label>
<input type="text" class="title" required placeholder="Title" name="title"/>
<label for="author">Author:</label>
<input type="text" class="author" required placeholder="Author" name="author"/>
<button type="submit" class="form-btn">Add</button>
</form>

```